\chapter{General Conclusions}
\label{chapterlabelConclusions}

\section{General summary}

In this thesis, I have explored the use of ChromoPainter on ancient DNA samples and present-day samples which contain sparsely genotyped markers. I evaluated the impact of coverage on all steps of the analysis pipeline, from imputation and phasing with GLIMPSE \cite{rubinacci2021efficient} to ChromoPainter and SOURCEFIND analysis, focussing on the trade-off between potential gains from leveraging haplotype information and potential reference panel bias. I then applied my findings to two novel and one publicly available dataset(s). 

In Chapter 2, I showed that the copyvectors of GLIMPSE imputed $\geq$0.5x downsamples show a high correspondence with the same sample at full coverage (Fig. \ref{fig:CP_correlation_allSamples_0.1x_0.5x_30x}), when painted using both ancient and present-day donors. 

Disappointingly, my several attempts to improve the performance of ChromoPainter on 0.1x and 0.5x samples were not successful, including filtering the SNPs used using different criteria (Section \ref{sec:Solutions}). This was surprising, as my work and that of others \cite{hui2020evaluating} showed that filtering SNPs on e.g. genotype probabilities could substantially reduce the overall fraction of incorrectly imputed genotypes. I also found evidence of bias towards the reference panel (Fig. \ref{fig:imputed_nonimputed_donation}), shown as excess donation from populations in the reference panel, and bias towards the reference sequence, as genotypes containing more reference alleles were imputed with greater accuracy (Table. \ref{tab:genotype_class_prop}). In part, these biases may be driven by various factors; for instance, although the sample size was small (n=5), my results also showed that ancient samples which are temporally/genetically closer to a reference panel of present-day individuals are imputed with a higher accuracy.

Using present-day samples, I also showed that you can gain haplotype information using sparsely genotyped data with (presumably) perfect information at each SNP. Specifically, individuals from Cornwall and Devon can be distinguished genetically with $>$90\% accuracy using only 1565 500-kb regions that contain $\approx$ 6.6 SNPs on average (i.e.\ $\approx$ 40,000 SNPs in total) (Table. \ref{table:windows_power_table_DevCorn}). A similar classification rate was found for distinguishing Mandenka from Senegal and Yoruba from Nigeria, with $>$90\% accuracy when using 1565 500-kb regions of $\approx$ 8 SNPs (Table \ref{table:windows_power_table_ManYor}). However, it appears current imputation approaches do not make reliable enough genotype calls on aDNA samples with $<$0.5x average coverage to provide many 500-kb windows with correctly called (and no incorrectly called) genotypes. Perhaps this is not surprising, as my exploration of 587 available ancient DNA samples revealed that genomes with 0.5x coverage have $<$1500 500-kb regions with 12 SNPs covered by even two reads (Figure \ref{fig:avg_good_windows}), making calling heterozygotes challenging (or impossible) throughout the genome.

In Chapter 3, I explored African ancestry in U.K. Biobank samples. Following from my Chapter 2 findings, I showed that it is possible to recover substantial haplotype information with only a fraction of the total number of SNPs usually used. Being able to use fewer SNPs in an analysis will allow different datasets to be merged and jointly analysed, opening up a larger array of questions to be answered, whilst also significantly reducing the computational footprint of an analysis. I found that in terms of fine-scale population assignment, performing imputation on non-European samples using a predominantly European reference panel (Haplotype Reference Consortium) biases ChromoPainter analyses towards reference populations (Fig \ref{fig:imputed_excess_copying_pops}), as does performing analysis in unlinked mode (Table \ref{table:TVD_copying}). Indeed, performing analysis on a majority imputed SNPs is more harmful for accuracy than using 70,000 SNPs in unlinked mode. This suggests that imputing to combine data from different SNP arrays, using the strategy I outlined in Chapter 3, may actually be more harmful than using a relatively small number ($<$100,000) of overlapping non-imputed SNPs when inferring fine-scale ancestry patterns.

My analyses showed that approximately 4\% of U.K. Biobank participants have at least 50\% African ancestry. Within this set of individuals, genetic ancestry from West Africa was very prevalent, consistent with historical events (Fig \ref{fig:haplotype_sharing_map_zoomed_II}). In particular, I found that there was over ten times the number of individuals with at least 50\% ancestry from Yoruba than there was the next most common ancestry. 

In Chapter 4, I analysed novel ancient DNA datasets from Bavaria with the samples spanning almost 7000 years of history. The analysis of ancient Bavarian samples recapitulated previous research which identified admixture events between early farmers and local hunter-gatherers, and the presence of steppe-related ancestry in the Late Neolithic. However, it also provided some less expected results, showing that samples with extremely different ancestries cohabited the same cave and the same time period. I also identified ancestry most closely related to Iron Age Italian source which arrived in Bavaria during the Iron Age, but was not present in the preceding Bronze Age. Future studies could increase the number of ancient sample sequenced from Bronze and Iron Age Bavaria in order to constrain the date the ancestry appears and source of origin. Finally, I showed that early Germanic and Slavic samples from the Middle Ages, which could not be distinguished using other ancient samples, showed strong genetic differences when analysed using present-day data (Fig. \ref{fig:germanic_slavic_HB_sharing}). Whilst I was able to identify structure down to the level of individuals countries, the lack of data from different regions in Germany meant that I was not able to determine whether there was fine-scale differential relatedness to the ancient samples for different German states. 

My final Chapter analysed the differences between Migration Era and Early Middle Age samples from Czechia. The data revealed that whilst different Migration Era samples displayed genetic affinities to a wide spectrum of other ancient and present-day populations, the Early Middle Age individuals were relatively more homogenous and broadly showed strong similarity to present-day Slavic speaking populations (Fig. \ref{fig:copymatrix_moderns_ancient_slavs}). However, fineSTRUCTURE analysis using present-day Slavic and non-Slavic speaking populations clearly showed that present-day Slavic speaking populations can be split into south-east and north-west clusters, with different ancient samples showing different affinities to each cluster. Lastly, I provided evidence that previously reported \cite{Hellenthal2014, MOSAIC_2019} signals of east-Asian admixture in eastern-European populations was also present in the Early Middle Age ancient Slavic samples (Fig. \ref{fig:EarlyMiddleAges_MOSAIC_3way_moderns_acoanc}. Although the five Migration Era samples represented an array of ancestries present in Czechia during that period, the sample size (n=3 at most) per sub-population was too low to reliably infer admixture events.  

\section{Recommendations}

My recommendations for analysing low coverage data are as follows:

\begin{enumerate}
\item If imputing samples using GLIMPSE and the 30x 1000 genomes reference, include samples with at least 0.5x mean coverage. Samples below this coverage (0.1x) show effects of coverage-related bias in copyvector estimation, SOURCEFIND analysis and positions on a PCA. 
\item When merging data from different genotyping arrays, it is preferable only to retain directly genotyped SNPs rather than imputing missing ones using a reference panel (e.g. using Eagle2 and HRC) This applies when the total number of directly genotyped SNPs is at least 45,000 (Fig. \ref{fig:Devon_Cornwall_TVD_reduced_assignment}).
\end{enumerate}

\section{Limitations of work and future avenues of research}


Firstly, I did not consider ancient samples from Africa. This is in part because of a lack of high coverage samples from Africa (Mota being the highest coverage at ~10x) and the vast majority of ancient DNA samples from western Eurasia. I expect results to differ when considering African samples. Africans harbour more diversity and have lower levels of background LD \cite{bosch2009decay} and thus would be expected to match shorter segments to other individuals. Imputation accuracy would likely be lower, in part because of less LD and higher genetic diversity, but also because less of the total proportion of genetic diversity is present in reference panels. Finally, the large population turnovers in Africa (e.g. the Bantu expansions) mean that many pre-Bantu ancient samples may harbour diversity that does not exist in present-day individuals. Therefore, it is possible that coverage greater than 0.5x may be necessary to accurately analyse African samples with ChromoPainter. 

I did not evaluate the effect of coverage on either fineSTRUCTURE or GLOBETROTTER analysis. This is because GLOBETROTTER struggles to identify admixture events in single samples and I only had a single downsample for each individual and level of coverage. To accurately estimate admixture events, segments of DNA within an individual copied from different populations need to be identified. Such segments may be particularly hard to identify in low coverage samples, as the segment boundaries may contain low-coverage SNPs.   

I didn't use the largest reference panel (HRC) to impute ancient samples, due to technical challenges in obtaining access to the data and so likely underestimate the potential accuracy of imputation on low coverage samples. Thus, future work should examine the scale of improvements in imputation accuracy when using extremely large reference panels. For example, plans to sequence the whole-genomes of 200,000 U.K. Biobank participants would provide an unparalleled resource to impute variants in ancient samples of western European ancestry. 

Whilst my attempt at incorporating genotype likelihoods into the ChromoPainter process only provided very modest improvements, the fact that this approach has been successful in other methods \cite{ngsLD, vieira2016estimating, Meisner719, Lipatov023374} suggests that in theory it should also be applicable to Chromosome painting. Future work on ChromoPainter could explore the reason why this did not work and suggest alternate ways in which to account for the uncertainty associated with low coverage data. Studies could also interrogate the performance of ChromoPainter on the range of coverages between 0.1-0.5x. Recent research has argued it is possible to infer ancestral relationships between samples as low as 0.1x in coverage, although only for particular applications such as demographic change \cite{colate}.

On the other hand, methodological advances in laboratory DNA extraction techniques, DNA enrichment and sequencing technologies and library preparation for ancient samples may mean that all samples can be sequenced to a high enough coverage that coverage-related effects are inconsequential. 




