\chapter{Introduction}

\section{Chromopainter and ancient DNA}

The first ancient DNA papers relied principally on statistical methods based upon allele-sharing or allele-frequencies. These methods, in particular f-statistics \cite{Green2010, Patterson2012, peter2016admixture} and Principle Component Analysis \cite{price2006principal}. 

The early studies of ancient DNA generally considered broad-scale questions about human history, such as the nature of human-archaic interactions or the spread of farming technology across Europe \cite{Lazaridis2014}. Due to the infancy of the field, sample sizes were small, as efficient methods to obtain genetic data from ancient samples, such as SNP capture arrays, had yet to be developed. These studies tended to look for genetic differences between populations which had been diverged for many generations; in such cases, powerful methods are not required. In the case of Lazaridis et al (2014), simply plotting Loschbour and Stuttgart/LBK on a PCA of modern individual showed they had substantially divergent ancestries, with Loschbour being most similar to present-day North-East Europeans, but falling well outside the variation of present-day Europeans, and Stuttgart clustering with present-day individuals from Tuscany. 

The first use of chromopainter on ancient DNA was in a seminal paper from Lazaridis et al (2014) \cite{Lazaridis2014}. The two samples, Loschbour and Stuttgart, we of high coverage and therefore imputation was not used.

Perhaps the first study to explicitly investigate chromopainter on ancient DNA was Martiniano et al (2017) \cite{Martiniano2017}. They first tested the accuracy of imputation on ancient DNA samples 

As sample sizes used in ancient DNA studies has rapidly increased to >100, more analyeses have incorporate haplotype-based methods. Each study typically carries out a small analysis to ensure that imputation in low-coverage ancient DNA samples is accurate. Antonio et al (2019) \cite{antonio2019ancient} analysed 127 ancient genomes of a mean coverage of 1x. To test imputation accuracy, they downsampled a single individual (NE1) to different levels of coverage and calculated the proportion of genotypes which matched the full coverage. However, this analysis was only performed on a single sample and the effect of imputation on the chromopainter process was not evaluated. 

More recently, ChromoPainter has been used to study aspects of archaic hominin ancestry in present-day humans \cite{JACOBS20191010, teixeira2021widespread}. 