% I may change the way this is done in a future version, 
%  but given that some people needed it, if you need a different degree title 
%  (e.g. Master of Science, Master in Science, Master of Arts, etc)
%  uncomment the following 3 lines and set as appropriate (this *has* to be before \maketitle)
% \makeatletter
% \renewcommand {\@degree@string} {Master of Things}
% \makeatother

\title{Harnessing haplotype sharing information from low coverage sequencing and sparsely genotyped data }
\author{Sam Morris}
\department{UCL Genetics Insitute}

\maketitle
\makedeclaration

\begin{abstract} % 300 word limit

Accounting for linkage information has been shown to enhance power to detect fine-scale population structure, particularly when considering recent shared ancestry. In particular, ChromoPainter has been shown to be a successful method at identifying shared haplotypes between samples. It has also been used widely on ancient DNA samples. However, coverage is an issue, and it is possible that analysing low-coverage samples may provide biased results, for example if using phasing and imputation algorithms to pre-process data. Whilst a small number of studies have tested the utility of using ChromoPainter on ancient DNA, none have tested a range of samples across different coverages, at all steps of the analysis pipeline. In this work, I assess the impact of coverage on each step of the ChromoPainter analysis pipeline. I show that bias can exist when exploring (e.g.) population structure using low-coverage samples, and I investigate a series of modifications and strategies to reduce the extent of this bias. I also address a related challenge of analysing haplotype information in sparsely genotyped data in present-day individuals, e.g. when analysing only variants that overlap multiple genotyping arrays. Using these findings, I infer fine-scale African ancestry in U.K. Biobank participants using a new reference panel of data from 349 African ethnolinguistic groups, demonstrating how imputation of sparsely genotyped samples can substantially harm the estimation of sub-continental ancestry. Furthermore, I analyse a novel ancient DNA dataset from Bavaria in order to determine the extent of continuity between the Late Neolithic and Iron Ages, as well as the age of east-west structure in Europe. I also analyse novel ancient DNA samples from Slavic-speaking regions, exploring the genetic relationship between samples from the Migration Era to the Early Middle Ages, and the signatures of these ancient populations in present-day Slavic speaking populations. Finally, I summarise my findings and recommend approaches for future work on haplotype-based studies using low-coverage or sparsely genotyped data. 

\end{abstract}

\begin{acknowledgements}
\noindent
I would like to thank the following people.

\noindent
First, my supervisor Garrett for his dedication and skill, not only in helping me through my PhD, but with all of the other small things along the way. 

\noindent
My Mum, Dad and sister for always supporting me and their voluntary proof-reading efforts.

\noindent
All the gang in office 212 who I so much fun with; Mislav, Lucy, Nancy, Magnifica, Arturo, Dave, Mike, Chris, Camus and all the others who came by, even for a bit. It was sad to be cut short, but I hope to see you all in the future. All of the LIDo guys as well, too many to name, who made being at UCL so enjoyable. 


\noindent
All the good folk at UCL Computer Science cluster, particularly Ed and David, for putting up with my poor cluster etiquette over the years. 

\noindent
Nadine for being the best administrator ever.

\noindent
Thank you to Pascal and Jay for looking after me when times were tough. 
\end{acknowledgements}


\section{Impact statement}

I intend that the work presented in this thesis will provide a foundation for other researchers who apply haplotype-based methods for the analysis of low coverage ancient DNA and sparsely genotyped. Specifically, the benchmarks I provide in Chapter 2 can be followed by scientists in order to perform reliable ancient DNA analyses. This is important, as many studies are now using the aforementioned techniques. I also hope that others will take over up work into adapting ChromoPainter for ancient DNA and make further improvements to the algorithm.Similarly, other researchers can use my results to make decisions on whether to retain a smaller number of SNPs, or impute missing ones, when merging datasets across multiple genotyping arrays. Given previous research has outlined the utility of accounting for haplotypes when accounting for population stratification in GWAS, my findings may be useful looking forward when such approaches become more common. 

My empirical work on ancient DNA in chapters 4 and 5 should a grounding for future work, much like the work I referenced in those sections aided me in understanding the historical and genetic context of the current research. For example, future studies may use these results to inform how they sample new ancient DNA samples. 

Outside of academia, I believe there is a fundamental benefit to learning about our history as a species, something which the study of ancient DNA has provided tools for in the past decade. Ancient DNA analysis remains a field with popular reach, so I hope my work will go a small way towards providing the public with interesting and scientifically valid findings.

I believe that exploring the ancestry of ethnic minorities within the U.K. Biobank can be of value to those individuals communities, particularly when they have been excluded from many similar kinds of analyses. Lastly, my work should also play a part in the inclusion of a more diverse array of ethnicities in association studies.  

\setcounter{tocdepth}{2} 
% Setting this higher means you get contents entries for
%  more minor section headers.

\tableofcontents
%\listoffigures
%\listoftables


