% I may change the way this is done in a future version, 
%  but given that some people needed it, if you need a different degree title 
%  (e.g. Master of Science, Master in Science, Master of Arts, etc)
%  uncomment the following 3 lines and set as appropriate (this *has* to be before \maketitle)
% \makeatletter
% \renewcommand {\@degree@string} {Master of Things}
% \makeatother

\title{Harnessing haplotype sharing information from low coverage sequencing and sparsely genotyped data }
\author{Sam Morris}
\department{UCL Genetics Insitute}

\maketitle
\makedeclaration

\begin{abstract} % 300 word limit
Accounting for linkage information has been shown to enhance power to detect fine-scale population structure, particularly when considering recent relationships. In particular, ChromoPainter has been shown to be a successful method at identifying shared haplotypes between samples. It has also been used widely on ancient DNA samples. However, coverage is an issue and it is possible that analysing low-coverage samples may provide biased results. Whilst a small number of studies have performed tests, none have tested a range of samples across different coverages, at all steps of the analysis pipeline. Therefore, in Chapter 2, I will assess the impact of coverage on each step of the ChromoPainter analysis pipeline. I will also investigate the possible causes of `coverage-related bias' and investigate a series of modifications to and strategies to reduce the extent of the bias. In Chapter 3, I will work on the related problem of low-density genotypes data in the context of exploring African ancestry within the U.K. Biobank and show that performing imputation can substantially harm the estimation of sub-continental ancestry. In Chapter 4, I will apply my findings from Chapter 2 to analyse a novel ancient DNA dataset from Bavaria in order to determine the extent of continuity between the Late Neolithic and Iron Ages, as well as the age of east-west structure in Europe. Similarly, Chapter 5 will also explore a novel ancient DNA dataset, this time from Slavic regions. By co-analysing the ancient samples with a set of ancient genomes from the literature, I will analyse the relationship between the Migration Era and Early Middle Ages, in addition to their relationship to present-day Slavic speaking populations. Finally, I will summarise my findings from the four chapters and recommend approaches for future work on haplotype-based studies using low-coverage or sparsely genotyped data. 


\end{abstract}

\begin{acknowledgements}
Thanks to all the good folk at UCL Computer Science cluster, particular Ed and David.

Thanks to my Mum and Dad. 

Thanks to all the gang in office 212, Mislav, Lucy, Nancy, Arturo, Dave, Mike, Chris, Camus. 

Thanks to Nadine. 

Thank you to Jay and Pascal for  
\end{acknowledgements}

\setcounter{tocdepth}{2} 
% Setting this higher means you get contents entries for
%  more minor section headers.

\tableofcontents
%\listoffigures
%\listoftables

